doc/content/reference/networking/_index.md

```diff
@@ -20,7 +20,15 @@ The following table lists the default ports used.
 | Management, data, events | gRPC | API key, token | 1884 | 8884 |
 | Management | HTTP | API key, token | 1885 | 8885 |
 | Backend Interfaces | HTTP | Custom | N/A | 8886 |
-| Basic Station LNS | HTTP | Auth Token, Custom | 1887 | 8887 |
+| LoRa Basic Station LNS | Web Sockets | Auth Token, Custom | 1887 | 8887 |
+
+### Enterprise-only Ports
```

Comment:
You can use the {{< distributions-inline "Enterprise" >}} shortcode inside the table to mark this, like https://github.com/TheThingsIndustries/lorawan-stack-docs/blob/master/doc/content/reference/branding/_index.md